bootstrap.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -ex
+# The -e option would make our script exit with an error if any command
+# fails while the -x option makes verbosely it output what it does
+
+# Install Pipenv, the -n option makes sudo fail instead of asking for a
+# password if we don’t have sufficient privileges to run it
```

Comment:
Please remove all those comments. Comments are there only to explain why the code was implemented in a certain way.